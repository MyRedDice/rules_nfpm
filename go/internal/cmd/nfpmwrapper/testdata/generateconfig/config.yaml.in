name:    test-package
version: {{ index .StableStatus `STABLE_VERSION` }}
arch:    amd64

description: |
    this is a package, it was generated at {{ index .VolatileStatus `BUILD_TIMESTAMP` }}.

files:
  "{{ index .Dependencies `//:example.txt`}}": /etc/example.txt
