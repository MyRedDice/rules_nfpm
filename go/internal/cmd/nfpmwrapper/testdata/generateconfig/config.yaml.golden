name:    test-package
version: v1.0.0.deadbeef
arch:    amd64

description: |
    this is a package, it was generated at 1591634962.

files:
  "example.txt": /etc/example.txt
